### ISSUE TITLE
issue with login tab in contact sheet

### ISSUE THREAD

**Issue by johbuch** [mautic/mautic#5808]
**Please DO NOT report security vulnerabilities here. Send them to security@mautic.com instead.**

What type of report is this:

| Q  | A
| ---| ---
| Bug report? | X
| Feature request? | 
| Enhancement? | 

## Description:
There is a bug on the social tab in the contact sheet.
If you import a contact and map the field twitter with a twitter url, then on the contact sheet the social tab is still on 0.
![image](https://user-images.githubusercontent.com/31535432/37413553-96a0b6fe-27a7-11e8-96db-799d77428fb9.png)
Then I check on my contact edition the twitter field. It is filled.
![image](https://user-images.githubusercontent.com/31535432/37413585-acb0c696-27a7-11e8-8c28-06eae3cc0b07.png)
I save and close.
Then the social tab has been updated and work
![image](https://user-images.githubusercontent.com/31535432/37413923-5932027c-27a8-11e8-9157-86b98d606a99.png)

In conclusion, with an import the social tab does not work so you need to edit the contact then save and close to see that is it finally displayed.

## If a bug:

| Q   | A
| --- | ---
| Mautic version | 2.12.2
| PHP version | 

### Steps to reproduce:
1. import a contact and map the twitter field with a twitter url
2. Go in the contact sheet
3. check the social tab and see that it is displayed 0
4. Edit the contact then just save and close
5. check the social tab and see that now it is dispayed 1 and the twitter account appears.
 
### Log errors: 

_Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable._
